CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ Most notably many tags start with a `t:` prefix (as in `topic:`), which categori
 In general *all issues are open for anyone working on them*, however if you're new to the project and looking for an issue
 that will be accepted and likely is a nice one to get started you should check out the following tags:
 
-- [community](https://github.com/akka/akka/labels/community) - which identifies issues that the core team will likely not have time to work on, or the issue is a nice entry level ticket. If you're not sure how to solve a ticket but would like to work on it feel free to ask in the issue about clarification or tips.
+- [help wanted](https://github.com/akka/akka/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) - which identifies issues that the core team will likely not have time to work on, or the issue is a nice entry level ticket. If you're not sure how to solve a ticket but would like to work on it feel free to ask in the issue about clarification or tips.
```

Comment:
"https://github.com/akka/akka/labels/help%20wanted" seems to work

Reply:
Even better :)